src/main/java/model/UserModel.java

```diff
@@ -113,5 +115,46 @@ public boolean resetPassword(String email) {
 		}
 		return result;
 	}
+	
+	public ArrayList<CommentBean> getCommentList(String id) {
+		ArrayList<CommentBean> result = new ArrayList<CommentBean>();
+		ResultSet data = daoFactory.getUserDao().getComments(Integer.parseInt(id));
+		
+		if(data != null) {
+			try {
+				while(data.next()) {
+					CommentBean comment = new CommentBean();
+					comment.setUserId(data.getInt("id_user"));
+					comment.setCommentatorId(data.getInt("id_commentator"));
+					comment.setCommentator(data.getString("commentator"));
+					comment.setComment(data.getString("comment"));
+					comment.setVote(data.getBoolean("vote"));
+					result.add(comment);
+				}
+			} catch (SQLException e) {
+				e.printStackTrace();
+			}
+		}
+		return result;
+	}
+	
+	public boolean resolveComment(Map<String, String[]> parameters) {
+		boolean result = false;
+		if(parameters != null && !parameters.isEmpty() && parameters.containsKey("commentator") && parameters.containsKey("user") && parameters.containsKey("action")) {
+			int commentatorId = Integer.parseInt(parameters.get("commentator")[0]);
+			int userId = Integer.parseInt(parameters.get("user")[0]);
+			boolean action = Boolean.parseBoolean(parameters.get("action")[0]);
```

Comment:
If the commentator or user does not actually contain integer, Integer.parseInt will throw a NumberFormatException which can be avoided with a simple regex or a try catch. It is also necessary to mention that Boolean.parseBoolean does not read 1 as true only "true" is true it might be good to have additional method to validate and parse boolean for other cases as well